gitops/binding_test.go

```diff
@@ -179,7 +179,6 @@ var _ = Describe("Gitops functions for managing Snapshots", Ordered, func() {
 		Expect(gitops.HaveAppStudioTestsSucceeded(hasSnapshot)).To(BeTrue())
 		bindingComponents := gitops.NewBindingComponents(hasSnapshot)
 		Expect(bindingComponents).NotTo(BeNil())
-		Expect(hasComp.Spec.Replicas).To(Equal(0))
```

Comment:
Why this change?

Reply:
The replicas are not automatically set to 0 in the newer versions of application-api (see this [commit](https://github.com/redhat-appstudio/application-api/commit/4b8b7924dfaf9981d479c6b4ca9f68b6e27368eb)) and it is now not defined by default. 
This is correct behavior because the replicas field is only there to override argoCD configuration if users wish to.